src/libcore/marker.rs

```diff
@@ -273,7 +273,11 @@ macro_rules! impls{
 /// even though it does not. This allows you to inform the compiler about certain safety properties
 /// of your code.
 ///
-/// Though they both have scary names, `PhantomData<T>` and "phantom types" are unrelated. 👻👻👻
+/// # A ghastly note 👻👻👻
+///
+/// Though they both have scary names, `PhantomData<T>` and 'phantom types' are related, but not
+/// identical. Phantom types are a more general concept that doesn't require `PhantomData<T>` to
```

Comment:
this is a tricky english thing. I forget the rule here, it depends if "Phantom types" or "concept" is the subject. I think you're right.